packages/phone-sensors/internal/phone-sensor.ts

```diff
@@ -0,0 +1,33 @@
+import BaseSensor = es.uji.geotec.backgroundsensors.sensor.BaseSensor;
+
+export enum PhoneSensor {
+  ACCELEROMETER = 'ACCELEROMETER',
+  GYROSCOPE = 'GYROSCOPE',
+  MAGNETOMETER = 'MAGNETOMETER',
```

Comment:
What do you think of having the value strings in lower case? This way, the casing is consistent with other enum values (and also it won't look like the sensor type is screaming to you when you read the captured records through the console 😂 #TOC)

Reply:
Hehe, I didn't notice that all the enums were in lowercase 😅